packages/common/src/store/account/sagas.ts

```diff
@@ -102,34 +113,27 @@ export function* fetchAccountAsync({ isSignUp = false }): SagaIterator {
   ])
   yield* call([libs, libs.setCurrentUser], {
     wallet,
-    userId: account.user_id
+    userId: user.user_id
   })
-  yield* put(signedIn({ account, isSignUp }))
+  yield* put(signedIn({ account: user, isSignUp }))
 }
 
-export function* cacheAccount(account: User) {
+export function* cacheAccount(account: AccountUserMetadata) {
+  const { user: accountUser, playlists: collections } = account
   const localStorage = yield* getContext('localStorage')
 
   yield put(
     cacheActions.add(Kind.USERS, [
-      { id: account.user_id, uid: 'USER_ACCOUNT', metadata: account }
+      { id: accountUser.user_id, uid: 'USER_ACCOUNT', metadata: account }
```

Comment:
oh oops, I think this should be `metadata: accountUser`!

Reply:
And also since this is now using the audiusQuery saga fetch stuff, I don't think we actually have to do this caching in this function. It will already be there.
This is actually what was causing the `null` entries.